functions/go/Makefile

```diff
@@ -153,3 +153,6 @@ func-push:
 docs-generate:
 	$(GOBIN)/mdtogo $(CURRENT_FUNCTION) $(CURRENT_FUNCTION)/generated --license=none --strategy=cmdDocs
 
+license:
```

Comment:
Please reconcile with https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/master/functions/go/Makefile#L108-L114

Reply:
I took another look, the `go-licenses` is more about analyzes a binary/package's dependencies and save those licenses to a specific directory, instead of add licenses for files, so it seems not the package we need.

Reply:
@zyy98 I think using more dedicated PR description and subject can help the confusion here.
@mengqiy We want to add a handy tool for ourselves that can add missing COPYRIGHT to each new files. This is not to verify the dependent libraries LICENSE.